Libs/vtkAddon/vtkAddonMathUtilities.cxx

```diff
@@ -116,3 +119,80 @@ void vtkAddonMathUtilities::GetOrientationMatrix(vtkMatrix4x4* source,
       }
     }
 }
+
+//----------------------------------------------------------------------------
+std::string vtkAddonMathUtilities::ToString(const vtkMatrix4x4* mat, const std::string delimiter, const std::string rowDelimiter)
```

Comment:
Could you please consolidate this with the to/from string conversion implemented in qSlicerTransformsModuleWidget::pasteTransform() and qSlicerTransformsModuleWidget::copyTransform()?

Probably the best is to have the string converters in vtkAddonMathUtilities and use that in qSlicerTransformsModuleWidget but we shouldn't lose any functionality. Conversion from string is not trivial, as we want to support 2x2, 3x3, 4x4 matrix with different separators (space, tab, comma, semicolon, etc) and ignore extra characters ( ()[] etc) to be convenient to both Python and Matlab users.